### ISSUE TITLE
Hardcoded frames

### ISSUE THREAD

**Issue by PetrosDavtyan** [scamps88/ASCalendar#6]
Is there any way to manage with frames without changing the library?

**Comment by scamps88** [comment]
not for now.
if you wish to modify the calendar size, it is specified in ASCalendarV.swift
(row 31)  bodyV = ASBodyV(frame: CGRect(x: 0, y: 40, width: **250**, height: **210**))

**Comment by Aku-n06** [comment]
No, for now the only way to  show the calendar is as a layer with fixed layout.
size and frame customisations will be the next implementation.
